crates/sui-core/src/authority/authority_per_epoch_store.rs

```diff
@@ -366,39 +363,6 @@ impl AuthorityPerEpochStore {
                 }
             })
             .collect();
-        // Insert epoch_start_configuration in the DB. This is used by unit tests
-        //
-        // Production code goes different path:
-        // (1) For the first epoch, this is inserted in the DB along with genesis checkpoint
-        // (2) For other epochs, this is updated when AuthorityPerEpochStore
-        // is initialized during epoch change
-        let epoch_start_configuration =
-            if let Some(epoch_start_configuration) = epoch_start_configuration {
-                assert_eq!(epoch_start_configuration.epoch_id(), epoch_id);
```

Comment:
I think we should preserve this `assert`

Reply:
fixed